apps/passport-client/components/screens/ZappScreens/ZappButton.tsx

```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-export const ZAPP_BUTTON_HEIGHT = 80;
+export const ZAPP_BUTTON_HEIGHT = 60;
```

Comment:
Why did we change this?

Reply:
the frogcryto btn got smaller so had to reduce size to remove extra bottom mergin under the btn